### PR TITLE
Added simple markers to represent start, end and stopovers

### PR DESCRIPTION
**Pull request by hnrqer**
#47

### CODE REVIEW

**Comment by hnrqer** [comment]
This is a small sample how markers would look like.
markers

Later we can choose better icons to replace the circles.

In this feature, only start (GREEN circle) and end (RED circle) markers are functional.
The blue circle I generated by the code by inserting a coordinate in between them.

ATM in the head solution, not even stopovers are being plotted, so when (and if) the solution #46 is merged, stopovers can also be easily added, as the function created supports more than one mark as input.

File: src/components/GreenNavMap.js [hunk 1]
```diff
@@ -131,6 +155,30 @@ export default class GreenNavMap extends Component {
...
+    const features = [];
+    markersCoordinate.forEach(function(el, i) {
+      let featureName = 'markerStopOver';
+      if (i == 0) {
+         featureName = 'markerStart';
+      } else if (i == markersCoordinate.length-1) {
+          featureName = 'markerEnd';
+      }
+
+      const circle = new ol.geom.Circle(el, 0.0001);
```
Comment (fabianbormann) [inline src/components/GreenNavMap.js]: 0.0001 sounds like a magic number 😉  .. could you add a constant or variable circleWidth = 0.0001; above?

File: src/components/GreenNavMap.js [hunk 2]
```diff
@@ -131,6 +155,30 @@ export default class GreenNavMap extends Component {
     }
   }
 
+  getFeaturesMarkers = (markersCoordinate) => {
+    const features = [];
+    markersCoordinate.forEach(function(el, i) {
```
Comment (fabianbormann) [inline src/components/GreenNavMap.js]: In my opinion "getMarkerFeatures" and "markerCoordinates" sound better..  
subsequently "el" should become "markerCoordinate" to increase the readability 😊

**Comment by hnrqer** [comment]
Recommend me nice icons, so that I can integrate in the solution

**Review by fabianbormann — CHANGES_REQUESTED** [review]
in the old polymer implementation (branch) we had already icons for waypoints. Could you improve your PR to use this old markers (or any other icon markers) ?

**Comment by hnrqer** [comment]
Ok! will make the changes and add the icons.

**Comment by hnrqer** [comment]
Can you please review the solution now? I created the .svg files representing the markers.

Check this out!

I used green for start point, red for end point and blue for stopovers. Can you give feedback regarding this idea?
marker icons

**Review by ar5had — COMMENTED** [review]
Looks Very nice.. :+1:  Can you choose a bit dull variant of these colors and shorten the size of icons? IMHO they are quite bright/warm and big.  :)

**Comment by hnrqer** [comment]
Thanks! The green color was actually the same from the GreenNav logo.

Do you have suggestions regarding the color pallete? I'm not sure the ones specified in http://greennav.io/development/design-style-guide/ would make it obvious what stop waypoint is, and what a stopover point is.

**Review by fabianbormann — APPROVED** [review]

**Pull request closed without merging**